alf/utils/dist_utils.py

```diff
@@ -345,9 +345,7 @@ def estimated_entropy(dist,
     detail.
 
     Args:
-        dist (tfp.distributions.Distribution): concerned distribution
-        seed (Any): Any Python object convertible to string, supplying the
-            initial entropy.
+        dist (torch.distributions.Distribution): concerned distribution
         assume_reparametrization (bool): assume the sample from continuous
```

Comment:
We don't need this. td.Distribution have `rsample()` and `sample()`. When rsample is available, we should use `rsample`, which is reparameterized sampling.